src/cata_tiles.h

```diff
@@ -385,7 +459,33 @@ class cata_tiles
          */
         bool nv_goggles_activated;
 
-
+        //pixel minimap cache methods
+        SDL_Texture_Ptr create_minimap_cache_texture(int tile_width, int tile_height);
+        void process_minimap_cache_updates();
+        void update_minimap_cache( const tripoint& loc, pixel& pix );
+        void prepare_minimap_cache_for_updates();
+        void clear_unused_minimap_cache();
+
+        std::map< std::string, minimap_cache_ptr> minimap_cache;
+
+        //persistent tiled minimap values
+        void init_minimap( int destx, int desty, int width, int height );
+        bool minimap_prep;
+        int minimap_minx;
+        int minimap_miny;
+        int minimap_maxx;
+        int minimap_maxy;
+        int minimap_tiles_range_x;
+        int minimap_tiles_range_y;
+        int minimap_tile_size_x;
+        int minimap_tile_size_y;
+        int minimap_tiles_x_limit;
+        int minimap_tiles_y_limit;
+        int minimap_drawn_width;
+        int minimap_drawn_height;
+        int minimap_border_width;
+        int minimap_border_height;
```

Comment:
All the x,y combinations should be stored in one member of type `point` (contains x and y).